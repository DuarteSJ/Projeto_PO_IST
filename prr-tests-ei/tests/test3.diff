*** tests/test3.out	2022-11-04 10:02:30.039831400 +0000
--- tests/test3.outhyp	2022-11-04 10:36:06.905398444 +0000
***************
*** 291,297 ****
  10 - Terminar comunicação activa
  11 - Mostrar comunicação em curso
  0 - Sair
! Escolha uma opção: Duração da comunicação (em minutos): Custo da comunicação: 20 unidades.
  OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
--- 291,297 ----
  10 - Terminar comunicação activa
  11 - Mostrar comunicação em curso
  0 - Sair
! Escolha uma opção: Duração da comunicação (em minutos): Custo da comunicação: 10 unidades.
  OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
***************
*** 386,392 ****
  3 - Operações de terminal
  0 - Sair
  Escolha uma opção: FANCY|111111|1|IDLE|2000|0|222222,333333,444444
! FANCY|222222|2|IDLE|1500|100|111111
  FANCY|333333|3|BUSY|0|0
  FANCY|444444|4|BUSY|0|4000
  GESTÃO DE TERMINAIS
--- 386,392 ----
  3 - Operações de terminal
  0 - Sair
  Escolha uma opção: FANCY|111111|1|IDLE|2000|0|222222, 333333, 444444
! FANCY|222222|2|IDLE|1500|90|111111
  FANCY|333333|3|BUSY|0|0
  FANCY|444444|4|BUSY|0|4000
  GESTÃO DE TERMINAIS
***************
*** 402,408 ****
  5 - Menu de consultas
  6 - Ver saldo global
  0 - Sair
! Escolha uma opção: Valores globais: 3500 (pagamentos),  4100 (dívidas).
  MENU PRINCIPAL
  1 - Abrir
  2 - Guardar
--- 402,408 ----
  5 - Menu de consultas
  6 - Ver saldo global
  0 - Sair
! Escolha uma opção: Valores globais: 0 (pagamentos),  0 (dívidas).
  MENU PRINCIPAL
  1 - Abrir
  2 - Guardar
***************
*** 428,434 ****
  VIDEO|6|222222|333333|1|20|FINISHED
  VIDEO|7|222222|333333|1|20|FINISHED
  VIDEO|8|222222|333333|1|20|FINISHED
! VIDEO|9|222222|333333|1|20|FINISHED
  VOICE|10|444444|222222|100|2000|FINISHED
  VOICE|11|444444|111111|100|2000|FINISHED
  VOICE|12|444444|333333|0|0|ONGOING
--- 428,434 ----
  VIDEO|6|222222|333333|1|20|FINISHED
  VIDEO|7|222222|333333|1|20|FINISHED
  VIDEO|8|222222|333333|1|20|FINISHED
! VIDEO|9|222222|333333|1|10|FINISHED
  VOICE|10|444444|222222|100|2000|FINISHED
  VOICE|11|444444|111111|100|2000|FINISHED
  VOICE|12|444444|333333|0|0|ONGOING
***************
*** 458,464 ****
  6 - Mostrar saldo de cliente
  0 - Sair
  Escolha uma opção: CLIENT|1|Manuel Pinheiro|103443|GOLD|YES|1|2000|0
! CLIENT|2|Pedro Pinheiro|103447|PLATINUM|YES|1|1500|100
  CLIENT|3|João Pinheiro|103125|NORMAL|YES|1|0|0
  CLIENT|4|John Pattenden|102546|NORMAL|YES|1|0|4000
  GESTÃO DE CLIENTES
--- 458,464 ----
  6 - Mostrar saldo de cliente
  0 - Sair
  Escolha uma opção: CLIENT|1|Manuel Pinheiro|103443|GOLD|YES|1|2000|0
! CLIENT|2|Pedro Pinheiro|103447|PLATINUM|YES|1|1500|90
  CLIENT|3|João Pinheiro|103125|NORMAL|YES|1|0|0
  CLIENT|4|John Pattenden|102546|NORMAL|YES|1|0|4000
  GESTÃO DE CLIENTES
