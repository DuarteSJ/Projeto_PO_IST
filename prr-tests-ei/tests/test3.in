4
3
111111
9
222222
VOICE
10
100
6
1
9
333333
VOICE
10
100
6
2
9
444444
VOICE
10
100
6
3
0
3
222222
9
111111
VIDEO
10
100
6
4
9
333333
VIDEO
10
1
9
333333
VIDEO
10
1
9
333333
VIDEO
10
1
9
333333
VIDEO
10
1
9
333333
VIDEO
10
1
0
3
444444
9
222222
VOICE
10
100
9
111111
VOICE
10
100
9
333333
VOICE
0
1
0
6
5
1
0
3
2
0
0
