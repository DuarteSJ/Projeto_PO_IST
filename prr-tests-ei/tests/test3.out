MENU PRINCIPAL
1 - Abrir
2 - Guardar
3 - Gestão de clientes
4 - Gestão de terminais
5 - Menu de consultas
6 - Ver saldo global
0 - Sair
Escolha uma opção: GESTÃO DE TERMINAIS
1 - Mostrar todos os terminais
2 - Registar terminal
3 - Operações de terminal
0 - Sair
Escolha uma opção: Número do terminal: OPERAÇÕES DO TERMINAL
1 - Ligar
2 - Colocar no silêncio
3 - Desligar
4 - Adicionar amigo
5 - Retirar amigo
6 - Pagamento
7 - Consultar saldo
8 - Enviar comunicação de texto
9 - Iniciar comunicação
11 - Mostrar comunicação em curso
0 - Sair
Escolha uma opção: Número do terminal: Tipo de comunicação a iniciar (VOICE, VIDEO): OPERAÇÕES DO TERMINAL
1 - Ligar
2 - Colocar no silêncio
3 - Desligar
4 - Adicionar amigo
5 - Retirar amigo
6 - Pagamento
7 - Consultar saldo
10 - Terminar comunicação activa
11 - Mostrar comunicação em curso
0 - Sair
Escolha uma opção: Duração da comunicação (em minutos): Custo da comunicação: 1000 unidades.
OPERAÇÕES DO TERMINAL
1 - Ligar
2 - Colocar no silêncio
3 - Desligar
4 - Adicionar amigo
5 - Retirar amigo
6 - Pagamento
7 - Consultar saldo
8 - Enviar comunicação de texto
9 - Iniciar comunicação
11 - Mostrar comunicação em curso
0 - Sair
Escolha uma opção: Número da comunicação: OPERAÇÕES DO TERMINAL
1 - Ligar
2 - Colocar no silêncio
3 - Desligar
4 - Adicionar amigo
5 - Retirar amigo
6 - Pagamento
7 - Consultar saldo
8 - Enviar comunicação de texto
9 - Iniciar comunicação
11 - Mostrar comunicação em curso
0 - Sair
Escolha uma opção: Número do terminal: Tipo de comunicação a iniciar (VOICE, VIDEO): OPERAÇÕES DO TERMINAL
1 - Ligar
2 - Colocar no silêncio
3 - Desligar
4 - Adicionar amigo
5 - Retirar amigo
6 - Pagamento
7 - Consultar saldo
10 - Terminar comunicação activa
11 - Mostrar comunicação em curso
0 - Sair
Escolha uma opção: Duração da comunicação (em minutos): Custo da comunicação: 500 unidades.
OPERAÇÕES DO TERMINAL
1 - Ligar
2 - Colocar no silêncio
3 - Desligar
4 - Adicionar amigo
5 - Retirar amigo
6 - Pagamento
7 - Consultar saldo
8 - Enviar comunicação de texto
9 - Iniciar comunicação
11 - Mostrar comunicação em curso
0 - Sair
Escolha uma opção: Número da comunicação: OPERAÇÕES DO TERMINAL
1 - Ligar
2 - Colocar no silêncio
3 - Desligar
4 - Adicionar amigo
5 - Retirar amigo
6 - Pagamento
7 - Consultar saldo
8 - Enviar comunicação de texto
9 - Iniciar comunicação
11 - Mostrar comunicação em curso
0 - Sair
Escolha uma opção: Número do terminal: Tipo de comunicação a iniciar (VOICE, VIDEO): OPERAÇÕES DO TERMINAL
1 - Ligar
2 - Colocar no silêncio
3 - Desligar
4 - Adicionar amigo
5 - Retirar amigo
6 - Pagamento
7 - Consultar saldo
10 - Terminar comunicação activa
11 - Mostrar comunicação em curso
0 - Sair
Escolha uma opção: Duração da comunicação (em minutos): Custo da comunicação: 500 unidades.
OPERAÇÕES DO TERMINAL
1 - Ligar
2 - Colocar no silêncio
3 - Desligar
4 - Adicionar amigo
5 - Retirar amigo
6 - Pagamento
7 - Consultar saldo
8 - Enviar comunicação de texto
9 - Iniciar comunicação
11 - Mostrar comunicação em curso
0 - Sair
Escolha uma opção: Número da comunicação: OPERAÇÕES DO TERMINAL
1 - Ligar
2 - Colocar no silêncio
3 - Desligar
4 - Adicionar amigo
5 - Retirar amigo
6 - Pagamento
7 - Consultar saldo
8 - Enviar comunicação de texto
9 - Iniciar comunicação
11 - Mostrar comunicação em curso
0 - Sair
Escolha uma opção: GESTÃO DE TERMINAIS
1 - Mostrar todos os terminais
2 - Registar terminal
3 - Operações de terminal
0 - Sair
Escolha uma opção: Número do terminal: OPERAÇÕES DO TERMINAL
1 - Ligar
2 - Colocar no silêncio
3 - Desligar
4 - Adicionar amigo
5 - Retirar amigo
6 - Pagamento
7 - Consultar saldo
8 - Enviar comunicação de texto
9 - Iniciar comunicação
11 - Mostrar comunicação em curso
0 - Sair
Escolha uma opção: Número do terminal: Tipo de comunicação a iniciar (VOICE, VIDEO): OPERAÇÕES DO TERMINAL
1 - Ligar
2 - Colocar no silêncio
3 - Desligar
4 - Adicionar amigo
5 - Retirar amigo
6 - Pagamento
7 - Consultar saldo
10 - Terminar comunicação activa
11 - Mostrar comunicação em curso
0 - Sair
Escolha uma opção: Duração da comunicação (em minutos): Custo da comunicação: 1500 unidades.
OPERAÇÕES DO TERMINAL
1 - Ligar
2 - Colocar no silêncio
3 - Desligar
4 - Adicionar amigo
5 - Retirar amigo
6 - Pagamento
7 - Consultar saldo
8 - Enviar comunicação de texto
9 - Iniciar comunicação
11 - Mostrar comunicação em curso
0 - Sair
Escolha uma opção: Número da comunicação: OPERAÇÕES DO TERMINAL
1 - Ligar
2 - Colocar no silêncio
3 - Desligar
4 - Adicionar amigo
5 - Retirar amigo
6 - Pagamento
7 - Consultar saldo
8 - Enviar comunicação de texto
9 - Iniciar comunicação
11 - Mostrar comunicação em curso
0 - Sair
Escolha uma opção: Número do terminal: Tipo de comunicação a iniciar (VOICE, VIDEO): OPERAÇÕES DO TERMINAL
1 - Ligar
2 - Colocar no silêncio
3 - Desligar
4 - Adicionar amigo
5 - Retirar amigo
6 - Pagamento
7 - Consultar saldo
10 - Terminar comunicação activa
11 - Mostrar comunicação em curso
0 - Sair
Escolha uma opção: Duração da comunicação (em minutos): Custo da comunicação: 20 unidades.
OPERAÇÕES DO TERMINAL
1 - Ligar
2 - Colocar no silêncio
3 - Desligar
4 - Adicionar amigo
5 - Retirar amigo
6 - Pagamento
7 - Consultar saldo
8 - Enviar comunicação de texto
9 - Iniciar comunicação
11 - Mostrar comunicação em curso
0 - Sair
Escolha uma opção: Número do terminal: Tipo de comunicação a iniciar (VOICE, VIDEO): OPERAÇÕES DO TERMINAL
1 - Ligar
2 - Colocar no silêncio
3 - Desligar
4 - Adicionar amigo
5 - Retirar amigo
6 - Pagamento
7 - Consultar saldo
10 - Terminar comunicação activa
11 - Mostrar comunicação em curso
0 - Sair
Escolha uma opção: Duração da comunicação (em minutos): Custo da comunicação: 20 unidades.
OPERAÇÕES DO TERMINAL
1 - Ligar
2 - Colocar no silêncio
3 - Desligar
4 - Adicionar amigo
5 - Retirar amigo
6 - Pagamento
7 - Consultar saldo
8 - Enviar comunicação de texto
9 - Iniciar comunicação
11 - Mostrar comunicação em curso
0 - Sair
Escolha uma opção: Número do terminal: Tipo de comunicação a iniciar (VOICE, VIDEO): OPERAÇÕES DO TERMINAL
1 - Ligar
2 - Colocar no silêncio
3 - Desligar
4 - Adicionar amigo
5 - Retirar amigo
6 - Pagamento
7 - Consultar saldo
10 - Terminar comunicação activa
11 - Mostrar comunicação em curso
0 - Sair
Escolha uma opção: Duração da comunicação (em minutos): Custo da comunicação: 20 unidades.
OPERAÇÕES DO TERMINAL
1 - Ligar
2 - Colocar no silêncio
3 - Desligar
4 - Adicionar amigo
5 - Retirar amigo
6 - Pagamento
7 - Consultar saldo
8 - Enviar comunicação de texto
9 - Iniciar comunicação
11 - Mostrar comunicação em curso
0 - Sair
Escolha uma opção: Número do terminal: Tipo de comunicação a iniciar (VOICE, VIDEO): OPERAÇÕES DO TERMINAL
1 - Ligar
2 - Colocar no silêncio
3 - Desligar
4 - Adicionar amigo
5 - Retirar amigo
6 - Pagamento
7 - Consultar saldo
10 - Terminar comunicação activa
11 - Mostrar comunicação em curso
0 - Sair
Escolha uma opção: Duração da comunicação (em minutos): Custo da comunicação: 20 unidades.
OPERAÇÕES DO TERMINAL
1 - Ligar
2 - Colocar no silêncio
3 - Desligar
4 - Adicionar amigo
5 - Retirar amigo
6 - Pagamento
7 - Consultar saldo
8 - Enviar comunicação de texto
9 - Iniciar comunicação
11 - Mostrar comunicação em curso
0 - Sair
Escolha uma opção: Número do terminal: Tipo de comunicação a iniciar (VOICE, VIDEO): OPERAÇÕES DO TERMINAL
1 - Ligar
2 - Colocar no silêncio
3 - Desligar
4 - Adicionar amigo
5 - Retirar amigo
6 - Pagamento
7 - Consultar saldo
10 - Terminar comunicação activa
11 - Mostrar comunicação em curso
0 - Sair
Escolha uma opção: Duração da comunicação (em minutos): Custo da comunicação: 20 unidades.
OPERAÇÕES DO TERMINAL
1 - Ligar
2 - Colocar no silêncio
3 - Desligar
4 - Adicionar amigo
5 - Retirar amigo
6 - Pagamento
7 - Consultar saldo
8 - Enviar comunicação de texto
9 - Iniciar comunicação
11 - Mostrar comunicação em curso
0 - Sair
Escolha uma opção: GESTÃO DE TERMINAIS
1 - Mostrar todos os terminais
2 - Registar terminal
3 - Operações de terminal
0 - Sair
Escolha uma opção: Número do terminal: OPERAÇÕES DO TERMINAL
1 - Ligar
2 - Colocar no silêncio
3 - Desligar
4 - Adicionar amigo
5 - Retirar amigo
6 - Pagamento
7 - Consultar saldo
8 - Enviar comunicação de texto
9 - Iniciar comunicação
11 - Mostrar comunicação em curso
0 - Sair
Escolha uma opção: Número do terminal: Tipo de comunicação a iniciar (VOICE, VIDEO): OPERAÇÕES DO TERMINAL
1 - Ligar
2 - Colocar no silêncio
3 - Desligar
4 - Adicionar amigo
5 - Retirar amigo
6 - Pagamento
7 - Consultar saldo
10 - Terminar comunicação activa
11 - Mostrar comunicação em curso
0 - Sair
Escolha uma opção: Duração da comunicação (em minutos): Custo da comunicação: 2000 unidades.
OPERAÇÕES DO TERMINAL
1 - Ligar
2 - Colocar no silêncio
3 - Desligar
4 - Adicionar amigo
5 - Retirar amigo
6 - Pagamento
7 - Consultar saldo
8 - Enviar comunicação de texto
9 - Iniciar comunicação
11 - Mostrar comunicação em curso
0 - Sair
Escolha uma opção: Número do terminal: Tipo de comunicação a iniciar (VOICE, VIDEO): OPERAÇÕES DO TERMINAL
1 - Ligar
2 - Colocar no silêncio
3 - Desligar
4 - Adicionar amigo
5 - Retirar amigo
6 - Pagamento
7 - Consultar saldo
10 - Terminar comunicação activa
11 - Mostrar comunicação em curso
0 - Sair
Escolha uma opção: Duração da comunicação (em minutos): Custo da comunicação: 2000 unidades.
OPERAÇÕES DO TERMINAL
1 - Ligar
2 - Colocar no silêncio
3 - Desligar
4 - Adicionar amigo
5 - Retirar amigo
6 - Pagamento
7 - Consultar saldo
8 - Enviar comunicação de texto
9 - Iniciar comunicação
11 - Mostrar comunicação em curso
0 - Sair
Escolha uma opção: Número do terminal: Tipo de comunicação a iniciar (VOICE, VIDEO): OPERAÇÕES DO TERMINAL
1 - Ligar
2 - Colocar no silêncio
3 - Desligar
4 - Adicionar amigo
5 - Retirar amigo
6 - Pagamento
7 - Consultar saldo
10 - Terminar comunicação activa
11 - Mostrar comunicação em curso
0 - Sair
Escolha uma opção: GESTÃO DE TERMINAIS
1 - Mostrar todos os terminais
2 - Registar terminal
3 - Operações de terminal
0 - Sair
Escolha uma opção: FANCY|111111|1|IDLE|2000|0|222222,333333,444444
FANCY|222222|2|IDLE|1500|100|111111
FANCY|333333|3|BUSY|0|0
FANCY|444444|4|BUSY|0|4000
GESTÃO DE TERMINAIS
1 - Mostrar todos os terminais
2 - Registar terminal
3 - Operações de terminal
0 - Sair
Escolha uma opção: MENU PRINCIPAL
1 - Abrir
2 - Guardar
3 - Gestão de clientes
4 - Gestão de terminais
5 - Menu de consultas
6 - Ver saldo global
0 - Sair
Escolha uma opção: Valores globais: 3500 (pagamentos),  4100 (dívidas).
MENU PRINCIPAL
1 - Abrir
2 - Guardar
3 - Gestão de clientes
4 - Gestão de terminais
5 - Menu de consultas
6 - Ver saldo global
0 - Sair
Escolha uma opção: CONSULTAS
1 - Mostrar todas as comunicações
2 - Mostrar comunicações feitas por um cliente
3 - Mostrar comunicações recebidas por um cliente
4 - Mostrar clientes sem dívidas
5 - Mostrar clientes com dívidas
6 - Mostrar terminais sem actividade
7 - Mostrar terminais com saldo positivo
0 - Sair
Escolha uma opção: VOICE|1|111111|222222|100|1000|FINISHED
VOICE|2|111111|333333|100|500|FINISHED
VOICE|3|111111|444444|100|500|FINISHED
VIDEO|4|222222|111111|100|1500|FINISHED
VIDEO|5|222222|333333|1|20|FINISHED
VIDEO|6|222222|333333|1|20|FINISHED
VIDEO|7|222222|333333|1|20|FINISHED
VIDEO|8|222222|333333|1|20|FINISHED
VIDEO|9|222222|333333|1|20|FINISHED
VOICE|10|444444|222222|100|2000|FINISHED
VOICE|11|444444|111111|100|2000|FINISHED
VOICE|12|444444|333333|0|0|ONGOING
CONSULTAS
1 - Mostrar todas as comunicações
2 - Mostrar comunicações feitas por um cliente
3 - Mostrar comunicações recebidas por um cliente
4 - Mostrar clientes sem dívidas
5 - Mostrar clientes com dívidas
6 - Mostrar terminais sem actividade
7 - Mostrar terminais com saldo positivo
0 - Sair
Escolha uma opção: MENU PRINCIPAL
1 - Abrir
2 - Guardar
3 - Gestão de clientes
4 - Gestão de terminais
5 - Menu de consultas
6 - Ver saldo global
0 - Sair
Escolha uma opção: GESTÃO DE CLIENTES
1 - Visualizar cliente
2 - Visualizar todos os clientes
3 - Registar cliente
4 - Activar notificações
5 - Desactivar notificações
6 - Mostrar saldo de cliente
0 - Sair
Escolha uma opção: CLIENT|1|Manuel Pinheiro|103443|GOLD|YES|1|2000|0
CLIENT|2|Pedro Pinheiro|103447|PLATINUM|YES|1|1500|100
CLIENT|3|João Pinheiro|103125|NORMAL|YES|1|0|0
CLIENT|4|John Pattenden|102546|NORMAL|YES|1|0|4000
GESTÃO DE CLIENTES
1 - Visualizar cliente
2 - Visualizar todos os clientes
3 - Registar cliente
4 - Activar notificações
5 - Desactivar notificações
6 - Mostrar saldo de cliente
0 - Sair
Escolha uma opção: MENU PRINCIPAL
1 - Abrir
2 - Guardar
3 - Gestão de clientes
4 - Gestão de terminais
5 - Menu de consultas
6 - Ver saldo global
0 - Sair
Escolha uma opção: 